api/camunda/camunda-openapi.yaml

```
@@ -441,6 +441,58 @@ paths:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
   /user-tasks/{userTaskKey}:
+    get:
+      tags:
+        - User task
+      summary: Return user task by a user task key.
```

```suggestion
      summary: Return user task by a user task key
```

:x: no periods in `summary`
owner: me in an upstream PR